src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -527,7 +527,7 @@ subroutine shelf_calc_flux(sfc_state_in, fluxes_in, Time, time_step_in, CS)
             if (wB_flux < 0.0) then
               ! The buoyancy flux is stabilizing and will reduce the turbulent
               ! fluxes, and iteration is required.
-              n_star_term = (ZETA_N/RC) * (hBL_neut * VK) / (ustar_h)**3
+              n_star_term = (ZETA_N/RC) * (hBL_neut) / (ustar_h)**3
```

Comment:
A very minor suggestion: Could this be rewritten to preserve order of operations? I would guess it's something like this: `((ZETA_N / RC) * hBL_neut) / ustar_h**3`

Reply:
Also @adcroft pointed out that there is a double division here which should be merged:
`(ZETA_N * hBL_neut) / (RC * ustar_h**3)`.

(Unless there is some significance to `ZETA_N/RC`, in which case a new variable should be introduced.)

Reply:
The double division on this line predated this PR and it is potentially answer changing, so I think that we can accept this as is and then deal with the double division separately.